app/lib/three_scale/redis_config.rb

```diff
@@ -56,5 +58,16 @@ def parse_sentinels(sentinels)
         parsed_sentinel
       end
     end
+
+    def load_default_ca_cert(conf)
+      return if conf.dig(:ssl_params, :ca_file).present? || conf.dig(:ssl_params, :ca_path).present?
+
+      cert_path = Rails.root.join('config/ca_cert.pem').to_s
```

Comment:
Another question. If customer has put this file into the OCP secret, will it end up and be found here automatically?

Reply:
It's not automatic unless implemented by the operator. Otherwise the user must edit the yamls to mount the certificate at `/opt/system/config` or `/opt/system-extra-configs`.

Reply:
For me that's the whole point of he earlier discussion - when the file is added to the secret, to be used automatically without further configuration. Otherwise it is unnecessary complicated for the users.

Reply:
What could we do from porta?

Reply:
We need to coordinate with operator folks how these can be mounted. For CA and cert files, we can put them in config maps I guess and have them mounted to default locations. For private keys though we need to use secrets and idk what is the best way to handle these.

Reply:
Why do we need to load a default CA cert in the first place? :thinking: 

I'd say - if nothing is provided in `:ca_file` or `:ca_path` explicitly, so be it? Does checking `config/ca_cert.pem` help in any way?...

Reply:
If there's no `:ca_file` or `:ca_path` it checks `config/ca_cert.pem` to load that one. If it doesn't exist, then it loads the system cert bundle in `/etc`.

The idea behind `config/ca_cert.pem` is there to provide a way for the user to use its own certificate effortlessly.

Reply:
But I think the effort is pretty much the same in both cases, just that the user will always need to set an env var for `:ca_file` or `:ca_path`. But at least it's clear where the cert should be mounted (wherever these env vars point to).

I am not sure we need to define some porta-specific CA cert location :shrug: 
But whatever, if you think it's useful, it's OK for me.

Reply:
I've been discussing with Eguz on how to integrate this TLS anc ACL with the operator, and we came to these two Jira issues:

https://issues.redhat.com/browse/THREESCALE-11020
https://issues.redhat.com/browse/THREESCALE-11021

Basically, porta and apisonator should limit to read the environment variables and it would be the operator who mounts the certificates and set the environment variables according to some new YAML keywords.

I made some changes to adapt to the new approach.